controller/auth-controller.js

```diff
@@ -0,0 +1,103 @@
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const User = require("../models/user");
+
+exports.login = async (req, res) => {
+  const {name, password} = req.body
+  const key = process.env.JWT_SECRET;
+
+  const user = await User.findOne({ userName: name });
+
+  if (!user) {
+    res.json({
+      err: "User dosent exist",
+    });
```

Comment:
What is the Response status here?
Please look into each response.

Reply:
updated